Automata_App/rabo_data/urls.py

```diff
@@ -18,7 +18,7 @@
 from rabo_data import views
 
 urlpatterns = [
-	url(r'^Interaction_Activitys/$', views.total, name='total'),
-    url(r'^Interaction_Activity/$', views.Interaction_ActivityList.as_view()),
-    url(r'^Interaction_Activity/(?P<pk>\w+)/$', views.Interaction_ActivityDetail.as_view()),
+	url(r'^Incident_Activitys/$', views.total, name='total'),
```

Comment:
should be: Incident_Activities

And while you're at it... you may as well make the urls lowercase :P